### PR TITLE
add timeout option

### PR DESCRIPTION
**Pull request by SamVerschueren**
First step in implementing https://github.com/sindresorhus/pageres/issues/250. Added it as an option here but we could hide it from pageres users.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -58,6 +58,13 @@ Default: `0`
 
 Delay capturing the screenshot. Useful when the site does things after load that you want to capture.
 
+##### timeout
+
+Type: `number` *(seconds)*  
+Default: `60`
+
+Number of seconds after which phantomjs aborts the request.
```
Comment (SamVerschueren) [inline readme.md]: Better suggestions?

Comment (kevva) [inline readme.md]: `phantomjs` => `PhantomJS`, otherwise it looks good to me.

**Comment by SamVerschueren** [comment]
:+1:

**Comment by kevva** [comment]
Would you mind fixing the conflicts and then we'll merge this :rocket:?

**Comment by sindresorhus** [comment]
:shipit:

**Comment by SamVerschueren** [comment]
@kevva Tests where still failing because of a port issues. Are you doing a fix commit then? The ports at [these](https://github.com/kevva/screenshot-stream/blob/master/test/test.js#L168-L169) lines should be bumped to `9004`.

**Pull request merged**